src/GraphQL/GraphQLExtensions.cs

```diff
@@ -30,7 +30,7 @@ public static bool IsCompositeType(this IGraphType type)
         public static bool IsLeafType(this IGraphType type)
         {
             var namedType = type.GetNamedType();
-            return namedType is ScalarGraphType || namedType is EnumerationGraphType;
```

Comment:
I think it is here for clarity. See comments below.

Reply:
i am not sure it adds any clarity. anyone can inherit from ScalarGraphType and we dont include them. perhaps @joemcbride can be tie breaker?

Reply:
I am not against removal. But then you need to remove all such uses from this file (or even solution).

Reply:
done